### PR TITLE
*: update vendor for pkgs of binlog

### PR DESCRIPTION
**Pull request by IANTHEREAL**
for pr #24

### CODE REVIEW

**Comment by IANTHEREAL** [comment]
@july2993 @WangXiangUSTC PTAL

**Comment by WangXiangUSTC** [comment]
LGTM

File: glide.yaml [hunk 1]
```diff
@@ -138,14 +138,3 @@ import:
...
-  subpackages:
-  - auth/authpb
-  - clientv3
-  - clientv3/concurrency
-  - etcdserver/api/v3rpc/rpctypes
-  - etcdserver/etcdserverpb
-  - mvcc/mvccpb
-  - pkg/logutil
-  - pkg/types
-- package: github.com/Shopify/sarama
```
Comment (july2993) [inline glide.yaml]: why delete this

Comment (IANTHEREAL) [reply glide.yaml]: where to use it？

Comment (IANTHEREAL) [reply glide.yaml]: oh, binlog client

Comment (IANTHEREAL) [reply glide.yaml]: add it again. some problems with our way of updating vendor. @july2993 PTAL

**Review by july2993 — APPROVED** [review]
LGTM

**Pull request merged**